src/nyc_trees/gulpfile.js

```diff
@@ -153,6 +153,7 @@ function browserifyTask(bundler) {
         })
         .bundle()
         .on('error', gutil.log.bind(gutil, 'Browserify Error'))
+        .on('error', process.exit.bind(process, 1))
```

Comment:
Can we put a condition around this so that `npm run watch` will keep running if Browserify fails to bundle the JS?

It might be a little tricky with the way the `watchify` / `browserify` tasks are setup, but it's nice to be able to leave the `watch` script running and not worry about accidentally saving a syntax error.